src/modules/metadata/hooks.js

```diff
@@ -33,3 +33,16 @@ export const useOverviewSummary = () => {
   }, [dispatch, metadataService]);
   return useSelector((state) => state.overviewSummary);
 };
+
+export const useProjectJsonSchemaTypes = () => {
+  const dispatch = useDispatch();
+  const metadataService = useService("metadata");
+  useEffect(() => {
+    dispatch(fetchExtraPropertiesSchemaTypes());
```

Comment:
does this action properly check if the metadata service exists before trying to execute?

Reply:
Yes it does